keps/sig-node/127-user-namespaces/README.md

```diff
@@ -486,6 +482,17 @@ inside the container:
 - `spec.initContainers[*].securityContext.runAsUser`
 - `spec.ephemeralContainers[*].securityContext.runAsUser`
 
+Further, validation will also be relaxed for pods in a baseline namespace:
+
+- `spec.containers[*].securityContext.capabilities`
+- `spec.initContainers[*].securityContext.capabilities`
+- `spec.ephemeralContainers[*].securityContext.capabilities`
+
+The validation for capabilities can be relaxed in a baseline pod because capabilities
+are user namespaced in the linux kernel, and any pod does not have a seccomp profile (as baseline
+pods are not required to) has access to the `unshare` syscall, allowing a user to create a user
```

Comment:
cc @tallclair @vinayakankugoyal for visibility / ack on the capabilities implications of pods without a seccomp profile that are in a user namespace